client/context/context.go

```diff
@@ -267,7 +273,7 @@ func (ctx CLIContext) PrintOutput(toPrint fmt.Stringer) (err error) {
 // GetFromFields returns a from account address and Keybase name given either
 // an address or key name. If genOnly is true, only a valid Bech32 cosmos
 // address is returned.
-func GetFromFields(from string, genOnly bool) (sdk.AccAddress, string, error) {
+func GetFromFields(from string, genOnly, legacySecretStore bool, input io.Reader) (sdk.AccAddress, string, error) {
```

Comment:
ditto, this API should not change. input should already exist in the context along with `--legacy` flag value.